humanoid_league_team_communication/scripts/show_team_comm.py

```diff
@@ -32,16 +34,14 @@
         SIDE: UNDEFINED
 """
 
-
-class TeamCommPrinter:
-
+class TeamCommPrinter(Node):
     def __init__(self):
-        rclpy.init(args=None)
+        super().__init__("team_comm")
         self.subscriber = self.create_subscription(TeamData, "team_data", self.data_cb, 1)
         self.team_data = {}
         for i in range(1, 5):
             self.team_data[i] = TeamData()
-            self.team_data[i].robot_id = -1
+            self.team_data[i].robot_id = i
```

Comment:
I'm not allowed to set this to `-1` anymore as it is defined as an unsigned int.
Should be fine, as it should be overwritten by the message callback anyway